app/services/to_cocina/description_mapper.rb

```diff
@@ -47,9 +47,9 @@ def params
 
     def note_params
       [].tap do |params|
-        if work_form.abstract.present?
+        if form.abstract.present?
```

Comment:
This is an example of a field that a work form will have, but a collection form won't, I think.

Reply:
Collections (at least from H2) often do have abstracts.